pkg/ctrl/controller_manager.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ctrl
+
+// DefaultControllerManager is the default ControllerManager.
+var DefaultControllerManager = &ControlleManager{}
+
+// ControllerManager initializes and starts Controllers.
+type ControlleManager struct {
```

Comment:
Could you put some documentation somewhere that explains how you expect the controller manager to be used? Are you expecting people to generally be writing multiple controllers that are bundled in the controller manager? It seems to me like a more-common use case will be to have a CRD with one controller associated with it, in which case having the controller manager seems like it adds some extra machinery that's not necessary.